packages/angular/cli/models/schematic-command.ts

```diff
@@ -309,7 +310,8 @@ export abstract class SchematicCommand<
               question.type = 'confirm';
               break;
             case 'list':
-              question.type = 'list';
+              const multiselect = !!definition.raw && (definition.raw as JsonObject)['multiselect'];
+              question.type = !!multiselect ? 'checkbox' : 'list';
```

Comment:
should be able to use `definition.multiselect` directly here.

Reply:
I was having trouble with this. At this point in the process, definition.multiselect is not present. It is only present in definition.raw. I am guessing there is some step where that is either not being set to the definition or is being lost. Would you have any insight as to where that might be?

Reply:
I made the requested changes minus this one... When testing this I am still having the issue where multiselect is not present at this point. I can see that it is present at some point before this, but  it is being stripped out at some point but I am having trouble identifying where.

Reply:
I think I have possibly figured out what is wrong. The built version of @angular/cli has a dependency on @angular-devkit/core, looking into the registry.js in the node_modules of @angular/cli I do not see the changes I have made, but looking in dist folder I do see the changes. So if I link the built @angular-devkit/core to the built @angular/cli this should show the changes I expect. Is there a better way to build the packages locally to test changes across multiple packages @clydin?

Reply:
Either way, this did work. So all changes have been addressed!